src/css/styles.scss

```diff
@@ -1,80 +1,90 @@
+$mainColor: THISTLE;
+$BKcolorHeadContainer: #484e52;
+$BKbody: #111f28;
```

Comment:
Try and avoid abbreviating your variable names. Remember the variable names are for us as developers, so keep them clear!